lagu123.fun,"HTTPSConnectionPool(host='m.lagu123.fun', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='m.lagu123.fun', port=443): Read timed out. (read timeout=2)"
lahiguera.net,"HTTPSConnectionPool(host='lahiguera.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.lahiguera.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
lahora.gt,"HTTPSConnectionPool(host='lahora.gt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='lahora.gt', port=443): Read timed out. (read timeout=2)"
lankainformation.lk,"HTTPSConnectionPool(host='lankainformation.lk', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='lankainformation.lk', port=80): Read timed out. (read timeout=2)"
lapaz.bo,"HTTPSConnectionPool(host='lapaz.bo', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f062ba64e50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='lapaz.bo', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f06275cc0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
larebajavirtual.com,"HTTPSConnectionPool(host='larebajavirtual.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='larebajavirtual.com', port=443): Read timed out. (read timeout=2)"
laroussecocina.mx,"HTTPSConnectionPool(host='laroussecocina.mx', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='laroussecocina.mx', port=443): Read timed out. (read timeout=2)"
lascondes.cl,"HTTPSConnectionPool(host='lascondes.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.lascondes.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
lasegunda.com,"HTTPSConnectionPool(host='lasegunda.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f062abaff70>, 'Connection to lasegunda.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='lasegunda.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f062982b7c0>, 'Connection to lasegunda.com timed out. (connect timeout=2)'))"
lasering.ee,"HTTPSConnectionPool(host='www.lasering.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lasering.ee', port=443): Read timed out. (read timeout=2)"
lavoiedushangrila.be,"HTTPSConnectionPool(host='lavoiedushangrila.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f062452c310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='lavoiedushangrila.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f062aa9f340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
lavprisvvs.dk,"HTTPSConnectionPool(host='www.lavprisvvs.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lavprisvvs.dk', port=443): Read timed out. (read timeout=2)"
lazimbana.com,"HTTPSConnectionPool(host='www.lazimbana.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lazimbana.com', port=443): Read timed out. (read timeout=2)"
lcb.lv,"HTTPSConnectionPool(host='lcb.lv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0626c9c850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='lcb.lv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0628a4ed60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ldb.lt,"HTTPSConnectionPool(host='ldb.lt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0623f77340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ldb.lt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0624504190>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ldblog.jp,"HTTPSConnectionPool(host='ldblog.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0627f82d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ldblog.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f06243c5520>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ldoceonline.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
le.ee,"HTTPSConnectionPool(host='le.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'le.ee' doesn't match either of '*.tll07.zoneas.eu', 'tll07.zoneas.eu'"")))","HTTPSConnectionPool(host='online.le.ee', port=443): Read timed out. (read timeout=2)"
lebanon24.com,"HTTPSConnectionPool(host='www.lebanon24.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lebanon24.com', port=443): Read timed out. (read timeout=2)"
led.go.th,"HTTPSConnectionPool(host='led.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f062a88be20>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='led.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0628f17a90>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
leggioggi.it,"HTTPSConnectionPool(host='leggioggi.it', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f06262599d0>, 'Connection to leggioggi.it timed out. (connect timeout=2)'))","HTTPConnectionPool(host='leggioggi.it', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f06266f4e20>, 'Connection to leggioggi.it timed out. (connect timeout=2)'))"
lekabobgrill.com,"HTTPSConnectionPool(host='lekabobgrill.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='lekabobgrill.com', port=80): Read timed out. (read timeout=2)"
lekari.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
lengua-e.com,"HTTPSConnectionPool(host='lengua-e.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f062b2df4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='lengua-e.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f062201e460>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
lenouvelliste.ch,"HTTPSConnectionPool(host='lenouvelliste.ch', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.lenouvelliste.ch', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
lerobert.com,"HTTPSConnectionPool(host='lerobert.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='lerobert.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
leto.gr,"HTTPSConnectionPool(host='leto.gr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0622608c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='leto.gr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0623e81040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
letras4u.com,"HTTPSConnectionPool(host='letras4u.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.letras4u.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
letssingit.com,"HTTPSConnectionPool(host='letssingit.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f06226087f0>, 'Connection to letssingit.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='letssingit.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f06273882b0>, 'Connection to letssingit.com timed out. (connect timeout=2)'))"
leumi.co.il,"HTTPSConnectionPool(host='leumi.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='leumi.co.il', port=80): Read timed out. (read timeout=2)"
lex-localis.info,"HTTPSConnectionPool(host='lex-localis.info', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0623d7b730>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='lex-localis.info', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0621cd07c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
lex.pl,"HTTPSConnectionPool(host='lex.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.lex.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
leychile.cl,"HTTPSConnectionPool(host='leychile.cl', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f062c0c2880>, 'Connection to leychile.cl timed out. (connect timeout=2)'))","HTTPConnectionPool(host='leychile.cl', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f06296f2790>, 'Connection to leychile.cl timed out. (connect timeout=2)'))"
lgov.hu,"HTTPSConnectionPool(host='lgov.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f062a743fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='lgov.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0621b41cd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
libelle-lekker.be,"HTTPSConnectionPool(host='libelle-lekker.be', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.libelle-lekker.be', port=443): Read timed out. (read timeout=2)"
liberation.fr,"HTTPSConnectionPool(host='liberation.fr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='liberation.fr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
librariaonline.ro,"HTTPSConnectionPool(host='www.librariaonline.ro', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.librariaonline.ro', port=443): Read timed out. (read timeout=2)"
librarything.es,"HTTPSConnectionPool(host='librarything.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'librarything.es' doesn't match either of 'www.librarything.de', 'www.librarything.es', 'www.librarything.fr', 'www.librarything.it', 'www.librarything.nl'"")))","HTTPSConnectionPool(host='librarything.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'librarything.es' doesn't match either of 'www.librarything.de', 'www.librarything.es', 'www.librarything.fr', 'www.librarything.it', 'www.librarything.nl'"")))"
lielvarde.lv,"HTTPSConnectionPool(host='lielvarde.lv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f06284d41c0>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPConnectionPool(host='lielvarde.lv', port=80): Read timed out. (read timeout=2)"
lifepharmacy.co.nz,"HTTPSConnectionPool(host='www.lifepharmacy.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lifepharmacy.co.nz', port=443): Read timed out. (read timeout=2)"
lifestyleblock.co.nz,"HTTPSConnectionPool(host='www.lifestyleblock.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.lifestyleblock.co.nz', port=443): Read timed out. (read timeout=2)"
lilafuge.hu,Exceeded 30 redirects.,Exceeded 30 redirects.
lineconomi.me,"HTTPSConnectionPool(host='lineconomi.me', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='lineconomi.me', port=443): Read timed out. (read timeout=2)"
lippu.fi,"HTTPSConnectionPool(host='www.lippu.fi', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.lippu.fi', port=80): Read timed out. (read timeout=2)"
lira.bg,"HTTPSConnectionPool(host='lira.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='lira.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
listafirme.ro,Exceeded 30 redirects.,Exceeded 30 redirects.
litoralpress.cl,"HTTPConnectionPool(host='www.litoralpress.cl', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='litoralpress.cl', port=80): Read timed out. (read timeout=2)"
livedoor.biz,"HTTPSConnectionPool(host='livedoor.biz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f062959d3a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='livedoor.biz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f061e2a15e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
livedoor.blog,"HTTPSConnectionPool(host='livedoor.blog', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f06218f8ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='livedoor.blog', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f06292cd190>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
livelib.ru,"HTTPSConnectionPool(host='www.livelib.ru', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.livelib.ru', port=443): Read timed out. (read timeout=2)"
llti.lt,"HTTPSConnectionPool(host='llti.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'llti.lt' doesn't match either of 'radius.llti.lt', 'ldap.llti.lt', 'kolab.llti.lt', 'smtp.llti.lt', 'pop3.llti.lt', 'imap.llti.lt', 'wmail.llti.lt', 'mail.llti.lt'"")))","HTTPSConnectionPool(host='wmail.llti.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
lmta.lt,"HTTPSConnectionPool(host='lmta.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='lmta.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
local360.co.il,"HTTPSConnectionPool(host='www.local360.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.local360.co.il', port=443): Read timed out. (read timeout=2)"
